app/models/partner.rb

```diff
@@ -228,6 +240,14 @@ def flipper_id
     "Partner;#{id}"
   end
 
+  def partials_to_show
+    displayable_partials || ALL_PARTIALS
+  end
+
+  def displayable_partials
```

Comment:
Could we put this as a private method since it is only being used in the `partial_to_show` right now?